workers/unity/Packages/com.improbable.gdk.tools/.DownloadCoreSdk/Program.cs

```diff
@@ -11,13 +11,14 @@ internal class Program
     {
         private static void Main(string[] args)
         {
-            if (args.Length == 0)
+            if (args.Length != 2)
             {
-                Console.Error.WriteLine("Specify the version of the CoreSdk to download.");
+                Console.Error.WriteLine("<path_to_spatial> <coresdk_version>");
```

Comment:
Shouldn't this say something like `Usage: <path_to_spatial> <coresdk_version>` ?